docs/core-concepts/editable-content-on-pages/custom-widgets.md

```diff
@@ -106,7 +106,7 @@ mkdir -p lib/modules/page-link-widgets/views
 ```
 
 ::: tip
-While it's good to get some experience making all the folders and files yourself, the `apostrophe-cli` pacakage that you installed earlier can easily create basic module structures for you from templates. You can learn more in the [`apostrophe-cli` README](https://github.com/apostrophecms/apostrophe-cli/blob/master/README.md).
+While it's good to get some experience making all the folders and files yourself, the `apostrophe-cli` package that you installed earlier can easily create basic module structures for you from templates. You can learn more in the [`apostrophe-cli` README](https://github.com/apostrophecms/apostrophe-cli/blob/master/README.md).
```

Comment:
Again I question this. I would almost reverse this logic.

Reply:
monkey see monkey do

Reply:
Not everything can be done with the CLI. Some understanding of the structure is both good and necessary.

Reply:
@localghost443 Could you flag this for discussion in the CLI branch? And I'm not 100% clear on what you're suggesting.